ctest-next/src/ast/constant.rs

```diff
@@ -0,0 +1,41 @@
+/// Represents a constant variable defined in Rust.
+pub struct Const {
```

Comment:
For basically every struct (public and not), it would be good to `#[derive(Clone, Debug)]`